pkg/common/rotationutil/rotationutil.go

```diff
@@ -13,6 +13,11 @@ func ShouldRotateX509(now time.Time, cert *x509.Certificate) bool {
 	return shouldRotate(now, cert.NotBefore, cert.NotAfter)
 }
 
+// X509Expired returns true if the given X509 cert has expired
+func X509Expired(now time.Time, cert *x509.Certificate) bool {
+	return !now.Before(cert.NotAfter)
```

Comment:
there's enough "nots" in here that while it's correct by my review, I'd like to see a small unit test (just passing in an expired and non-expired cert)

Reply:
Added a test, and simplified the condition.